src/web/app/components/question-responses/grouped-responses/grouped-responses.component.html

```diff
@@ -2,10 +2,10 @@
     *ngIf="responses.length && responses[0].allResponses.length">
   <div class="col-md-3">
     <div>
-      {{ isGrq ? 'To' : 'From' }}: {{ responses[0].allResponses[0][isGrq ? 'recipient' : 'giver']}}
+      {{ fromToTop }}
```

Comment:
A thing to note here is that you do not need to combine the string. It would make thing more complicated.. At least, you will find it is hard to give a proper name to it.

My suggestions is that you only need to append the suffix:
```
{{ isGrq ? 'To' : 'From' }}: {{ responses[0].allResponses[0][isGrq ? 'recipient' : 'giver']}} {{ teamInfo }}
```